ttnn/cpp/ttnn/operations/matmul/device/multi_core_reuse_mcast_2d_optimized/bmm_op_multi_core_reuse_mcast_2d_optimized.cpp

```diff
@@ -187,7 +188,7 @@ operation::ProgramWithCallbacks create_program_mcast_in0_in1(
     CoreRange all_cores(
         {(std::size_t)start_core_x, (std::size_t)start_core_y},
         {(std::size_t)start_core_x + num_cores_c - 1, (std::size_t)start_core_y + num_cores_r - 1});
-    const auto& cores = grid_to_cores(all_cores.start, all_cores.end, true);
+    const auto& cores = grid_to_cores(all_cores.start_coord, all_cores.end_coord, true);
```

Comment:
how was this compiling?

Reply:
I am assuming you were looking at a single commit change (81791a2); the initial commit added the line:
`const auto& cores = grid_to_cores(all_cores.start, all_cores.end, true);`
but it was done some time ago, and then in the subsequent commit this is corrected to reflect latest updates on main.